{{cookiecutter.project_slug}}/scripts/prepare.sh

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env sh
```

Comment:
Please remove this script, as the installation would be part of the Dockerfiles.